src/main/scala/org/stormenroute/mecha/package.scala

```diff
@@ -432,4 +432,65 @@ package object mecha {
     def default(v: =>T): Input.Query[T] = Input.default(query)(v)
   }
 
+  private[mecha] object GitIgnore {
+    import java.nio.file.{FileSystems, Paths, Files, Path}
+    import java.nio.file.StandardOpenOption._
+    import scala.collection.JavaConversions._
+
+    val log = MechaLog.Println
+
+    def ignore(ignoreFilePath: Path, toAdd: String) = {
+      log.info(s"Adding '$toAdd' to '${ignoreFilePath.toString}'...")
+      writeFile(ignoreFilePath, add(readFile(ignoreFilePath), toAdd))
+    }
+
+    private def readFile(file: Path): Seq[String] = {
+      Files.readAllLines(file)
```

Comment:
You should specify a charset here, since our builders are still running on JDK 7.